docs/command-reference/server-management/command.md

```diff
@@ -69,65 +56,16 @@ Examples:
-* **movablekeys:** the _first key_, _last key_, and _step_ values don't determine all key positions.
-  Clients need to use `COMMAND GETKEYS` or [key specifications][td] in this case.
-  See below for more details.
-* **no_auth:** executing the command doesn't require authentication.
-* **no_async_loading:** the command is denied during asynchronous loading (that is when a replica uses disk-less `SWAPDB SYNC`, and allows access to the old dataset).
-* **no_mandatory_keys:** the command may accept key name arguments, but these aren't mandatory.
-* **no_multi:** the command isn't allowed inside the context of a [transaction](https://redis.io/topics/transactions).
-* **noscript:** the command can't be called from [scripts](https://redis.io/topics/eval-intro) or [functions](https://redis.io/topics/functions-intro).
-* **pubsub:** the command is related to [Redis Pub/Sub](https://redis.io/topics/pubsub).
-* **random**: the command returns random results, which is a concern with verbatim script replication.
-  As of Redis 7.0, this flag is a [command tip][tb].
+* **noscript:** the command can't be called from [scripts](https://redis.io/topics/eval-intro).
 * **readonly:** the command doesn't modify data.
-* **sort_for_script:** the command's output is sorted when called from a script.
-* **skip_monitor:** the command is not shown in `MONITOR`'s output.
-* **skip_slowlog:** the command is not shown in `SLOWLOG`'s output.
-  As of Redis 7.0, this flag is a [command tip][tb].
-* **stale:** the command is allowed while a replica has stale data.
 * **write:** the command may modify data.
 
-### Movablekeys
-
-Consider `SORT`:
-
-```
-1) 1) "sort"
-   2) (integer) -2
-   3) 1) write
-      2) denyoom
-      3) movablekeys
-   4) (integer) 1
-   5) (integer) 1
-   6) (integer) 1
-   ...
-```
-
-Some Redis commands have no predetermined key locations or are not easy to find.
-For those commands, the _movablekeys_ flag indicates that the _first key_, _last key_, and _step_ values are insufficient to find all the keys.
-
-Here are several examples of commands that have the _movablekeys_ flag:
-
-* `SORT`: the optional _STORE_, _BY_, and _GET_ modifiers are followed by names of keys.
-* `ZUNION`: the _numkeys_ argument specifies the number key name arguments.
-* `MIGRATE`: the keys appear _KEYS_ keyword and only when the second argument is the empty string.
-
-Redis Cluster clients need to use other measures, as follows, to locate the keys for such commands.
-
-You can use the `COMMAND GETKEYS` command and have your Redis server report all keys of a given command's invocation.
-
-As of Redis 7.0, clients can use the [key specifications](#key-specifications) to identify the positions of key names.
-The only commands that require using `COMMAND GETKEYS` are `SORT` and `MIGRATE` for clients that parse keys' specifications.
-
-For more information, please refer to the [key specifications page][tr].
+// TODO: should we describe our internal flags?
```

Comment:
should we describe things like reverse-mapping and no-autojournal? I guess they shouldn't be user-facing

Reply:
I left no description and instead filed an issue https://github.com/dragonflydb/dragonfly/issues/861